src/plugin/pelias/geocoder/search.js

```diff
@@ -1,152 +1,145 @@
-goog.provide('plugin.pelias.geocoder.Search');
+goog.module('plugin.pelias.geocoder.Search');
 
-goog.require('goog.log');
-goog.require('goog.log.Logger');
-goog.require('ol.Feature');
-goog.require('ol.format.GeoJSON');
-goog.require('ol.geom.Point');
-goog.require('os.alert.AlertEventSeverity');
-goog.require('os.alert.AlertManager');
-goog.require('os.config.Settings');
-goog.require('os.extent');
-goog.require('os.geo');
-goog.require('os.search.AbstractUrlSearch');
-goog.require('plugin.pelias.geocoder.AttrResult');
-goog.require('plugin.pelias.geocoder.Result');
+const log = goog.require('goog.log');
+const GeoJSON = goog.require('ol.format.GeoJSON');
+const olProj = goog.require('ol.proj');
+const MapContainer = goog.require('os.MapContainer');
+const Settings = goog.require('os.config.Settings');
+const osExtent = goog.require('os.extent');
+const osMap = goog.require('os.map');
+const osProj = goog.require('os.proj');
+const AbstractUrlSearch = goog.require('os.search.AbstractUrlSearch');
+const {ID} = goog.require('plugin.pelias.geocoder');
+const Result = goog.require('plugin.pelias.geocoder.Result');
```

Comment:
Nit that isn't specific to here: the requires aren't being sorted by the script. We haven't been doing this consistently, but I did like it before.

Reply:
I agree, I need to make a sort that works with assigned require statements. I also need to find a way to do this automatically in an editor. It's less straightforward now that the lines don't naturally sort since we want to sort by the required module name.